tests/completers/test_bash_completer.py

```diff
@@ -164,3 +164,13 @@ def test_quote_handling(command_context, completions, lprefix):
         isinstance(comp, RichCompletion) and not comp.append_closing_quote
         for comp in bash_completions
     )  # make sure the completer handles the closing quote by itself
+
+
+@skip_if_on_darwin
+@skip_if_on_windows
+def test_bash_completer_empty_prefix():
+    context = CompletionContext(
+        CommandContext(args=(CommandArg("apt"),), arg_index=1, prefix="")
+    )
+    bash_completions, bash_lprefix = complete_from_bash(context)
+    assert {"install", "show"}.issubset(bash_completions)
```

Comment:
can't the subprocess call be mocked? without apt (like my system, it will fail). Ultimately the bash-completion invokes subprocess right. You can refer test_gitstatus.py.

Reply:
It won't really help since we're testing the integration between xonsh and bash.. If we mock bash we won't really be testing anything.
Maybe can you find some other program that exists broadly instead of apt?

Reply:
Yes that will do. one of coreutils.

Reply:
Changed to git

Reply:
Changed to git